.github/workflows/integration-test.yml

```diff
@@ -9,12 +9,12 @@ jobs:
     steps:
       - name: Download OpenSearch Core
         run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.1/latest/linux/x64/builds/opensearch/dist/opensearch-min-1.3.1-linux-x64.tar.gz
+          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/builds/opensearch/dist/opensearch-min-1.3.2-linux-x64.tar.gz
```

Comment:
```suggestion
          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-1.3.2-linux-x64.tar.gz
```

Reply:
@cliu123 Thanks for the suggestions. I updated the changes in new commit.